lib/firefox/webdriver/firefoxDelegate.js

```diff
@@ -23,12 +27,39 @@ class FirefoxDelegate {
       : 'none';
     this.firefoxConfig = options.firefox || {};
     this.options = options;
+    this.android = null;
     // We keep track of all alias and URLs
     this.aliasAndUrl = {};
+    // Variables for android power testing
+    this.screenBrightnessMode = 0;
+    this.screenBrightness = 127;
   }
 
   async onStart() {
     this.hars = [];
+    if (isAndroidConfigured(this.options)) {
+      this.android = new Android(this.options);
+    }
+
+    if (this.android && this.options.androidPower) {
+      log.info('Initializing power testing');
+
+      // Disable adaptive brightness
+      this.screenBrightnessMode = await this.android._runCommandAndGet(
```

Comment:
I'm wondering maybe this is something we always wanna do, even  when we don't run the power function to make sure all tests run with the same settings?

Reply:
Screen settings shouldn't affect technical/visual performance numbers very much (if at all) so it likely won't have an impact on anything that isn't a power test. I'm happy to move this to a more general setup step though if you'd like though.

Reply:
No lets keep it that way then :)